rfcs/text/0001-rfc-process.md

```diff
@@ -70,26 +86,66 @@ Where necessary, existing RFCs can be revised using the same process (a proposed
 
 ## Detailed Design
 
+**ADDITION BEGINS**
+
+### Drafting RFCs
+
+#### **Authors**:
+
+There are two types of RFCs that may be proposed: ***informational*** RFCs and ***non informational*** RFCs. Non informational RFCs are expected to be linked directly to an existing ticket in the DCP Zenhub board which will usually be tagged as a Spike. Informational RFCs are more speculative and need not directly reference an open ticket in the DCP Zenhub board.
+
+To inform the DCP Community about the development of an RFC, with the intention of forming collaborations between people thinking about similar problems, it is recommended that a draft of an RFC be first published once the Problem Statement and Authors have been established. One possible process that is recommended is as follows:
```

Comment:
_possible process_ makes it sound like there are other ways to create a RFC.

Reply:
Good catch on wording. Made it more straightforward.